CMakeLists.txt

```diff
@@ -69,9 +69,6 @@ set(VERSION_MAJOR 2)
 set(VERSION_MINOR 4)
 set(VERSION_PATCH 2)
 
-# add the SFML header path
-include_directories(${CMAKE_CURRENT_SOURCE_DIR}/include)
```

Comment:
to be "exportable", this needs to be defined on targets, not globally, thus https://github.com/SFML/SFML/pull/1335/files#diff-031c6047a0f4c227a313b7b2f3b77219R143